server/services/users/authentication_service.py

```diff
@@ -68,6 +68,11 @@ def login_user(osm_user_details, redirect_to, user_element='user') -> str:
 
         osm_id = int(osm_user.attrib['id'])
         username = osm_user.attrib['display_name']
+        try:
+            # get gravatar profile picture file name
+            user_picture = osm_user.find('img').attrib['href'].split('/avatar/')[1].split('?s=')[0]
```

Comment:
**You can mostly ignore the below, I realized that I just need to redo my Docker instance to enable the new API:**

This didn't seem to work for my user: 
This code: <img width="178" alt="image" src="https://user-images.githubusercontent.com/283343/62203546-68030380-b359-11e9-89f0-17c9b2a77037.png">
OSM:
<img width="110" alt="image" src="https://user-images.githubusercontent.com/283343/62203567-6e917b00-b359-11e9-9404-cfb7e6d3d2b0.png">
3.4 TM:
<img width="270" alt="image" src="https://user-images.githubusercontent.com/283343/62203797-e069c480-b359-11e9-8ccd-a9cf0b379fb5.png">

I did see one entry in Network tab that looks like it found my gravatar image but I was not able to reproduce it.

Here's the element on OSM.org:
`<img class="user_thumbnail_tiny" alt="" src="https://www.gravatar.com/avatar/fcf1c2b12340d8be939430bcd011e15d.jpg?s=100&amp;d=https%3A%2F%2Fwww.openstreetmap.org%2Fassets%2Favatar_large-afe7442b856c223cca92b1a16d96a3266ec0c86cac8031269e90ef93562adb72.png" width="25" height="25">`

Using the dispatcher in Redux dev tools:
```
{
type: 'SET_PICTURE',
userPicture:'fcf1c2b12340d8be939430bcd011e15d.jpg?s=100&amp;d=https%3A%2F%2Fwww.openstreetmap.org%2Fassets%2Favatar_large-afe7442b856c223cca92b1a16d96a3266ec0c86cac8031269e90ef93562adb72.png'
}
``` 
resulted in:
<img width="136" alt="image" src="https://user-images.githubusercontent.com/283343/62205475-82d77700-b35d-11e9-822c-28c993235009.png">

and here's the tag for what I'm seeing with this code:
https://gist.github.com/thadk/79a9b26d77285cc2b2ab6a9dcd2baeaa